docs/basics/1d_hist.rst

```diff
@@ -266,6 +267,12 @@ The function :func:`make_hist() <plothist.histogramming.make_hist>` returns a a
 
 The examples below show how to create a histogram with a category axis in ``boost_histogram`` and plot it with ``plothist``.
 
+.. note::
+    When we create the histograms in the examples below, we use the argument ``storage=bh.storage.Weight()``.
+    This is because functions in ``plothist`` assume histograms with a storage supporting weighted data in which the variance of each bin is tracked.
```

Comment:
This is because the functions in ``plothist'`` assume histograms with a storage that supports weighted data, where the variance of each bin is tracked.